frontend/src/components/shared/theme-toggle.module.css

```
@@ -0,0 +1,41 @@
+}
+
+.themeToggle:hover {
+  background: var(--bg-active);
+  color: var(--ink-1);
+  border-color: var(--line-strong);
+}
+
+.themeToggle svg {
+  width: var(--sp-4);
+  height: var(--sp-4);
+  stroke: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)